.github/workflows/go.yaml

```diff
@@ -0,0 +1,49 @@
+name: Go
```

Comment:
is a GH thing, or can with call it secret-agent. Go isn't a great name.

Reply:
I used a template they had for Go. I think we can name it something else

Reply:
actually, I think we should leave this with "go". We can add different ones later on with different names depending on the type of verification they run. 

Anyway, all actions in this project will be related to `secret-agent` so that'd be implicit